.github/workflows/pull-request.yml

```diff
@@ -23,3 +23,15 @@ jobs:
                   ACCESS_KEY_ID: ${{ secrets.ACCESS_KEY_ID }}
                   SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }}
                   REGION: ${{ secrets.REGION }}
+            - name: Ballerina Test
+              uses: ballerina-platform/ballerina-action/@nightly
+              with:
+                  args:
+                      test --test-report --code-coverage --coverage-format=xml
+              env:
+                  CLIENT_ID: ${{ secrets.CLIENT_ID }}
+                  CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
+                  REFRESH_TOKEN: ${{ secrets.REFRESH_TOKEN }}
```

Comment:
Can't we merge this with the above step?

Reply:
Fixed